src/axios.ts

```diff
@@ -44,13 +44,17 @@ export function makeAxiosInstanceWithRetry(
   return makeAxiosInstance(config, {
     retries,
     retryCondition: (error) => {
-      const isGraphQLEndpoint = !isNil(error.config.url)
-        && error.config.url.endsWith('graphql')
-        && error.config.method === 'post';
+      const isGraphQLEndpoint =
+        !isNil(error.config?.url) &&
+        Boolean(error?.config?.url.endsWith('graphql')) &&
+        error?.config?.method === 'post';
```

Comment:
it's complaining  `error.config.url` might be undefined.
and i have to add `?` and also force possibly `undefined` to be false with `Boolean()`. lmk if there's a cleaner way

Reply:
maybe try:

const isGraphQLEndpoint = 
   error?.config?.url?.endsWith('graphql') && 
   error.config.method === 'post';

Reply:
ok. yea. seems like the first `isNil` is not needed. but we might still need to do `Boolean` to force undefined to false. let me try